### ISSUE TITLE
Taking damage pose

### ISSUE THREAD

**Issue by IoriBranford** [ElizaWy/LPC#10]
I propose a "taking damage" pose such as:
![lpc-character-damage-pose](https://github.com/user-attachments/assets/cafd9c88-cb9b-4cd7-bcb9-0ef13dd53dbb)

If I commissioned an artist for initial sheets of this, would you be open to a PR? I'm also open to other pose ideas and recommendations of artists to commission. Thanks!

**Comment by jrconway3** [comment]
Eliza hasn't been active in a while, so its hard to say. I think a lot of LPC sprites have started off as commissions, you just have to make sure you make an agreement with whatever artist you commission to make sure they're willing to release the assets under certain OGA-supported licenses.

Specifically if you want it in LPC Revised it must be licensed under OGA-BY, which is almost identical to Creative Commons' CC-BY license. Other licenses are available as well at OGA (opengameart.org) if you are fine with it not being part of LPC Revised.

That being said, any assets you contribute (even commissioned assets) would be useful for the LPC project as a whole, too.

**Comment by ElizaWy** [comment]
Nono, I'm around. I've been having some health issues and I've paused my Patreon for the last few months, since I haven't been able to make the same kind of commitment to the project as I'd like. Chronic illness is a bitch. :/

Getting death and damage sprites are pretty high priority, after I finish the new slash animations.

**Comment by jrconway3** [comment]
Sorry to hear you've been ill lately. Do you mind other contributions? I know this is mostly your project, but Lori was offering to contribute, and I might be willing to contribute some stuff I've been working on, too. I just haven't seen you around, so I've been focusing on ULPC stuff mostly.

**Comment by jrconway3** [comment]
"Getting death and damage sprites are pretty high priority"

Yeah I'm not fully happy with the old death animations, but they're better than nothing, and I'm not too great at handling bases myself... I hadn't even thought of "damage" animations until seeing this issue get added here.

**Comment by ElizaWy** [comment]
I'm going to start with the old death animation (legacy death)-- the important thing is to standardize the animation guides so we can have standard clothing applied to it, within the set. Right now the pantlines and such wobble horribly.

As to contributions-- I absolutely grab everything that fits the license and isn't too far off stylistically! I admit, I haven't been keeping up OGA lately. There was drama, OGA has no 'block' features, and I'm avoiding the whole mess.

**Comment by jrconway3** [comment]
Well I've only been back to OGA since early this year (around March), so IDK if there was drama or not. I even subbed to your Patreon... right before you temporarily put it on pause lol.

In any case, I'm still working on assets myself over at Sanderfrenken's fork of the ULPC and occasionally updating things. Been also working with your assets as well. Some of my modifications might be able to be reversed back to here as well if you're interested. I prefer working with OGA-BY, so all assets I make/update also maintain the OGA-BY license (as long as the original asset is capable of doing so, and I do prefer working with OGA-BY assets first).


I'd love to see some new base animations, but for now I'm just trying to fill out the existing frames with as many assets as possible, while also creating some custom assets so I can make my own characters. Its a slow process.

**Comment by IoriBranford** [comment]
Masculine
![male-head](https://github.com/user-attachments/assets/e0e047ed-bc99-4837-9f05-2095f3a80ebe)
![male-body](https://github.com/user-attachments/assets/bb4495f5-87b1-481f-89c6-5da57ad8a5b3)

Feminine
![female-head](https://github.com/user-attachments/assets/605733d4-b7c4-4c3a-9b5f-54f7d07a7e3c)
![female-body](https://github.com/user-attachments/assets/a4e987d3-ddd9-41dd-9977-2950ca334b61)

Credit goes to Salvador Oblea Jr (joblea). If these look good, I'll submit a PR with all the palette variants when I have a chance, but not sure when that might be. They'll be in the Emotes unless otherwise suggested.

**Comment by jrconway3** [comment]
Huh, not too bad. You do have permission to release this under OGA-BY, right?

Honestly, I hadn't considered putting it under emotes, but looking it over, I do think it'd fit better under emotes. It does make more sense that way. Its just unfortunate than when adding new base frames, it means there's tons of work to add assets to said frames. So even with the correct license I'd probably still wait to add it to ULPC as well.


Anyway, again, please confirm the artist has allowed you to release this under OGA-BY. It won't be able to be added to LPC Revised at all without the OGA-BY license.

**Comment by IoriBranford** [comment]
Yes, I asked about OGA-BY before commissioning and he said "It's ok"

**Comment by jrconway3** [comment]
So I assume licensing attribution for this asset would go something like this:

```"Damage Base Model" by Salvador Oblea Jr. (joblea), Commissioned by Iori Branford. Licensed under OGA-BY 3.0```

**Comment by IoriBranford** [comment]
`Iori Branford` but yeah 👍

**Comment by jrconway3** [comment]
My bad, thanks for the correction. I thought it was a lowercase "L" lol.

**Comment by Bibisik** [comment]
Good afternoon, can you help me figure out how to use all this? How can I create and upload a character?
